python/ray/exceptions.py

```diff
@@ -133,17 +135,37 @@ def __str__(self):
         """Format a RayTaskError as a string."""
         lines = self.traceback_str.strip().split("\n")
         out = []
+        # We try not to print the error line if it is in worker
+        # because that contains the internal implemenation of Ray
+        # which is unnecessary for users.
         in_worker = False
         for line in lines:
             if line.startswith("Traceback "):
                 out.append(f"{colorama.Fore.CYAN}"
                            f"{self.proctitle}"
                            f"{colorama.Fore.RESET} "
-                           f"(pid={self.pid}, ip={self.ip})")
+                           f"(pid={self.pid}, ip={self.ip}"
+                           f", label={self.label})")
             elif in_worker:
                 in_worker = False
-            elif "ray/worker.py" in line or "ray/function_manager.py" in line:
+            elif ("ray/worker.py" in line
+                  or "ray/_private/function_manager.py" in line
+                  or "_private/client_mode_hook.py" in line):
```

Comment:
Nice fix :+1:

Reply:
What if we just considered all _private as internal?

Reply:
Sounds good. Also added tracing_helper.py